src/components/Button/index.tsx

```diff
@@ -253,13 +257,22 @@ function Button(
                     </View>
                     {shouldShowRightIcon && (
                         <View style={[styles.justifyContentCenter, large ? styles.ml2 : styles.ml1, iconRightStyles]}>
-                            <Icon
-                                src={iconRight}
-                                fill={iconFill ?? (success || danger ? theme.textLight : theme.icon)}
-                                small={small}
-                                medium={medium}
-                                large={large}
-                            />
+                            {!isSplitButton ? (
+                                <Icon
+                                    src={iconRight}
+                                    fill={iconFill ?? (success || danger ? theme.textLight : theme.icon)}
+                                    small={medium}
```

Comment:
why was it set like that? is there any specific reason? Right now I need to add small icon handling and I'll adjust it in this way:
> small={small}
> medium={medium}
> large={large}

may it broke something?
@nkdengineer